src/Angor/Client/Pages/Settings.razor

```diff
@@ -444,4 +596,32 @@
 
         NavMenuState.NotifyStateChanged();
     }
+
+    private void ChangeNetwork()
+    {
+        if (!confirmWipe)
+        {
+            showConfirmWipeMessage = true;
+            return;
+        }
+
+        _clientStorage.WipeStorage();
```

Comment:
so for now a wipe will delete all the database? this is ok but I could later change it so that the wallet will not be wiped but all the chain specific data (utxo, project, portfolio etc...)